python-package/lightgbm/compat.py

```diff
@@ -90,6 +90,19 @@ class DataFrame(object):
 except ImportError:
     GRAPHVIZ_INSTALLED = False
 
+"""datatable"""
+try:
+    from datatable import DataTable
+    DT_INSTALLED = True
```

Comment:
@guolinke Don't you mind to rename this variable to `DATATABLE_INSTALLED`, for the consistency with other variables (for example, there are `PANDAS_INSTALLED` but not `PD_INSTALLED`). Also, `DT` is a little bit confusing: sometimes `dt` is used for `datetime`.